### DISCUSSION TITLE
Cannot resolve `react` when using `@headlessui/react` with `parcel`

### DISCUSSION THREAD

**Discussion by hood** [tailwindlabs/headlessui#1534] [Help]
**What package within Headless UI are you using?**
@headlessui/react

**What version of that package are you using?**
v0.3.1

**What browser are you using?**
Chrome

**Reproduction URL**
~

**Describe your issue**
Cannot resolve `react` when using `@headlessui/react` with `parcel`. Getting the following error:
```
Parcel encountered errors


@parcel/core: Failed to resolve 'react' from '/home/me/node_modules/@headlessui/react/dist/hooks/use-computed.js'

[/home/me/node_modules/@headlessui/react/dist/hooks/use-computed.js:1:26](http://localhost:1234/__parcel_launch_editor?file=%2Fhome%2Fme%2Fnode_modules%2F%40headlessui%2Freact%2Fdist%2Fhooks%2Fuse-computed.js%3A1%3A26)
> 1 | }from"react";import{useIsoMorphicEffect as f}from"./use-iso-morphic-effect
>   |      ^^^^^^^
  2 |
```

**Comment by adamwathan** [comment]
Hey! If you're going to open an issue, please provide a reproduction, we require it to be able to help. Without one, we are totally in the dark and have to start with questions like "did you install `react`?" which is what my first guess would be, but also feels very much like contacting customer support and being asked "is your computer plugged in?" and that's not the type of experience we want to give people.
